changelogs/CHANGELOG-v2.5.rst

```diff
@@ -20,6 +20,7 @@ Release Summary
 
 Bugfixes
 --------
+- correct example code for AWS lightsail
```

Comment:
This file is auto generated.  Please create the changelog entry in `changelogs/fragments`